backends-common/rabbitmq/src/main/java/org/apache/james/backend/rabbitmq/RabbitMQConfiguration.java

```diff
@@ -35,16 +35,20 @@
 
     public static class ManagementCredentials {
 
-        static final char[] DEFAULT_PASSWORD = "guest".toCharArray();
         static final String DEFAULT_USER = "guest";
+        static final String DEFAULT_PASSWORD_STRING = "guest";
```

Comment:
Are the DEFAULT_PASSWORD_STRING & DEFAULT_USER still required?

Reply:
I fixed. They are required. those constants are used in tests

Reply:
Can we then locate them where appropriate in tests?